build.gradle

```diff
@@ -24,6 +24,12 @@ allprojects {
 	configurations.all {
 		resolutionStrategy.cacheChangingModulesFor 0, "minutes"
 	}
+
+	repositories {
```

Comment:
This is a temporary override of the regular snapshot repo (defined [here](https://github.com/spring-projects-experimental/spring-pulsar/blob/b9710b947a258925e80a42393a0976a07a4327a7/buildSrc/src/main/groovy/io/spring/gradle/convention/RepositoryConventionPlugin.groovy#L52)) and will be removed once `0.2.0` is released, correct? This allows us to move forward and get a green CI run now.

Reply:
Exactly. That's why the PR is in Draft mode atm